rockcraft.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+package-repositories:
+  - components: [main]
+    key-id: 0DCFCA5547B19D2A6099506096B3EE5F29111145
+    key-server: https://packages.wazuh.com/key/GPG-KEY-WAZUH
+    suites: [stable]
+    type: apt
+    url: https://packages.wazuh.com/4.x/apt/
+
+name: wazuh-server
+summary: Wazuh server
+description:  Wazuh server OCI image
+version: "1.0"
+base: ubuntu@22.04
+build-base: ubuntu@22.04
```

Comment:
Any reason not to start with 24.04 for this?

Reply:
Updated